src/config/laravel-analytics.php

```diff
@@ -11,17 +11,12 @@
      * Path to the client secret json file. Take a look at the README of this package
      * to learn how to get this file.
      */
-    'service_account_credentials_json' => storage_path('app/laravel-google-analytics/service-account-credentials.json'),
+    'service_account_credentials_json' => env('GOOGLE_APPLICATION_CREDENTIALS', storage_path('app/laravel-google-analytics/service-account-credentials.json')),
```

Comment:
Yeah, it is up to discussion, I added it because google is using as a convention here (find GOOGLE_APPLICATION_CREDENTIALS): https://github.com/google/google-api-php-client/blob/master/UPGRADING.md
But maybe it not necessary in the context of this library